From: svn://uclibc.org/trunk/buildroot/toolchain/gcc/4.1.2/302-c99-snprintf.patch

Path: 302-c99-snprintf.patch
Name: 302-c99-snprintf.patch
URL: svn://uclibc.org/trunk/buildroot/toolchain/gcc/4.1.2/302-c99-snprintf.patch
Repository Root: svn://uclibc.org
Repository UUID: 69ca8d6d-28ef-0310-b511-8ec308f3f277
Revision: 21884
Node Kind: file
Schedule: normal
Last Changed Author: sjhill
Last Changed Rev: 10249
Last Changed Date: 2005-05-05 05:17:15 +0200 (Thu, 05 May 2005)
Text Last Updated: 2008-03-27 14:45:05 +0100 (Thu, 27 Mar 2008)
Checksum: 49145ad9e849728301685e5950cca6b6

---
 libstdc++-v3/include/c_std/std_cstdio.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: gcc-4.1.2/libstdc++-v3/include/c_std/std_cstdio.h
===================================================================
--- gcc-4.1.2.orig/libstdc++-v3/include/c_std/std_cstdio.h
+++ gcc-4.1.2/libstdc++-v3/include/c_std/std_cstdio.h
@@ -143,7 +143,7 @@ namespace std
   using ::vsprintf;
 }
 
-#if _GLIBCXX_USE_C99
+#if _GLIBCXX_USE_C99 || defined(__UCLIBC__)
 
 #undef snprintf
 #undef vfscanf
